controllers/users_controller.js

```diff
@@ -3,133 +3,118 @@ const utils = require('../utils/util')
 const jwt = require('jwt-simple')
 const cfg = require('../config')
 
-module.exports.findAllUsers = (req, res) => {
-    user.findAllUsers().subscribe(
-        users =>
-            res.status(200).json({
-                error: false,
-                users: users
-            }),
-        err =>
-            res.status(404).json({
-                error: true,
-                users: []
-            })
-    )
+module.exports.findAllUsers = async (req, res) => {
+    try {
+        const users = await user.findAllUsers()
+        res.status(200).json({
+            error: false,
+            data: users
+        })
+    }catch (e) {
```

Comment:
Maybe use **eslint** to normalize codebase?

Reply:
I'm working on that rn, thinking of using [airbnb/javascript](https://github.com/airbnb/javascript)

Reply:
They have something like that at Qlik, ill see what they use and suggest that

Reply:
airbnb style requires semicolons, eww

Reply:
Lol I think we should use semicolons everywhere 😛

Reply:
nooooo, I think we should use [JS Standard Style](https://standardjs.com/)